docs/UserGuide.md

```diff
@@ -211,98 +234,47 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
-|-----|------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `find-all-tag t/remote t/full time` <br><br> **Output:** Shows a list of employees who are both full-time and work remotely |
-
-* **Acceptable Values:**
-  * Tags: Any predefined tags available in the address book (e.g. remote, contract, fullTime)
-  * Only full words will be matched, e.g. “remote” will not match “remotely”
-* **Expected Output on Success:**
-  * GUI Changes: A list of employees who match some specified tags will be shown in a dedicated interface section
-  * Message shown to the user: "[Number of matched employees] employees matched: "
-* **Expected Output on Failure:**
-  * `Invalid command format!` - Missing parameters
-  * `Tags names only allows alphanumeric characters, spaces, and dashes.` - Invalid parameter inputs
-
-### Finding employees with at least one tag in a group of tags: `find-some-tag`
-
-
-* **What It Does:**
-  * Filters and displays individuals from a dataset who match at least one of the specified tags (case-sensitive). It provides flexibility by showing records that meet any of the criteria provided.
-
-* **Format:**
-  * `find-some-tag t/TAG [t/MORE_TAGS]...`
-* **Examples:**
-
-| S/N | Command information                                                                                                                                                           |
-|-----|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `find-some-tag t/fullTime t/remote ` <br><br> **Output:** Shows a list of employees who either full-time or work remotely or both full-time and working remotely |
-
-* **Acceptable Values:**
-  * Tags: Any predefined tags available in the address book (e.g. remote, contract, fullTime)
-  * Only full words will be matched, e.g. “remote” will not match “remotely”
-* **Expected Output on Success:**
-  * GUI Changes: A list of employees who match some of the specified tags will be shown in a dedicated interface section
-  * Message shown to the user: "[Number of matched employees] employees matched: "
-* **Expected Output on Failure:**
-  * `Invalid command format!` - Missing parameters
-  * `Tags names only allows alphanumeric characters, spaces, and dashes.` - Invalid parameter inputs
-
-### Locating an employee in HRMate by name: `find`
+### Finding employees
+#### Finding employees by name: `find`
+  1. Type in the following [command](#glossary) in the [command box](#glossary) `find NAME....` where `NAME` is the name of the employee to search (case-insensitive) and `NAME...` representing one or more names. Note that only names containing the whole word are matched.
```

Comment:
I think we can rewrite these instructions to be more user-centric (adopt a more personal/friendly tone) + make it clear that the user should type out keywords that they want matched, as the normal expectation is that the search will return only names that match the entire phrase

Reply:
I think it returns names with either words